commands/motor/MotorPositionControl.java

```diff
@@ -74,21 +74,25 @@ protected void execute() {
 		catch (InvalidSensorException e) {
 			cancel();
 			if (fallbackCommand != null) {
-				fallbackCommand.start();
+				fallbackCommand.schedule();
 			}
 		}
 	}
 
 	@Override
-	protected boolean isFinished() {
+	public boolean isFinished() {
 		return false;
 	}
 
 	@Override
-	protected void end() {}
-
-	@Override
-	protected void interrupted() {
+	public void end(boolean interrupted) {
 		LogKitten.d("MotorPositionControl interrupted");
```

Comment:
Same as before, we don't want it to log "interrupted" when the command wasn't actually interrupted and just ended normally.